app/tests/integration/nimbus/utils/filter_expression.js

```diff
@@ -0,0 +1,25 @@
+Components.utils.import("resource://gre/modules/components-utils/FilterExpressions.jsm");
+Components.utils.import("resource://gre/modules/components-utils/ClientEnvironment.jsm");
+
+// Targeting
+
+async function remoteSettings(arguments) {
+
+    const TargetingContext = ChromeUtils.import(
+        "resource://messaging-system/targeting/Targeting.jsm"
+    )
+
+    _experiment = JSON.parse(arguments[1])
```

Comment:
Not sure if you want to add any JavaScript comments around here on what `arguments[1]` (and `arguments[0]` indexes represent below). Ie: One's an expression, one's the data/context or whatever.